tests/containers/bci_prepare.pm

```diff
@@ -81,6 +80,26 @@ sub activate_virtual_env {
     assert_script_run('source bci/bin/activate');
 }
 
+sub check_ltss_container {
+    my ($host_dist, $host_vers, $sp, $cont_vers, $cont_image) = @_;
+    # work with all inputs
+    return unless ($host_vers && $host_dist && $sp && $cont_vers && $cont_image);
+    # skip non-ltss
+    return if ("$host_vers-$sp" =~ m/^15-[67]/ || $cont_image !~ /ltss/i);
+    # check version match
+    my $hdd = get_var('HDD_1');
+    record_info("LTSS validation", $hdd);
+    die "OUT SCOPE: LTSS containers $cont_vers on $host_dist $host_vers host test excluded."
+      unless ($host_dist =~ /^sle/ && $host_vers =~ /^1[25]/ && $host_vers eq substr($cont_vers, 0, 2));
+    # check LTSS activation
+    my $status = q(SUSEConnect -s | jq -Mr '.[] | select(.identifier == "SLES-LTSS") | .subscription_status');
```

Comment:
I think the whole PR can re reduced to a SLES version check and this command.

@m-dati would you care to simplify the PR please?

Reply:
Please, could you elaborate about 'simplify', being this logic at different levels of checks?
This is a check that rules is satisfied or stop run in the negative case and each previous 'return' or 'die' is propaedeutic to next level of checks till final return.

Reply:
```
if (get_var('CONTAINER_IMAGE_TO_TEST') =~ "ltss") {
    validate_script_output('SUSEConnect -s | jq -Mr ".[] | select(.identifier == \"SLES-LTSS\") | .subscription_status"', qr/ACTIVE/, fail_message => "Host requires LTSS subscription for LTSS container");
}
```

Reply:
You could even make a one-liner out of this.

Reply:
But, lines #88 and #93 also are needed to be verified, not olny the ltss part: In your proposal those tests are missing.

Reply:
They are not needed because on those versions, the LTSS extension is not available anyways.

Reply:
Plus: Removing those version checks means we also do not need to edit the code in the future, when 15-SP6 or 15-SP7 become LTSS as well. Your check via `SUSEConnect` still works and does not require future changes.

Reply:
ok